src/main/java/ru/skypro/homework/service/impl/AuthServiceImpl.java

```diff
@@ -1,47 +1,41 @@
 package ru.skypro.homework.service.impl;
 
-import org.springframework.security.core.userdetails.User;
+import lombok.RequiredArgsConstructor;
 import org.springframework.security.core.userdetails.UserDetails;
 import org.springframework.security.crypto.password.PasswordEncoder;
-import org.springframework.security.provisioning.UserDetailsManager;
 import org.springframework.stereotype.Service;
+import ru.skypro.homework.dto.MyDatabaseUserDetails;
 import ru.skypro.homework.dto.Register;
+import ru.skypro.homework.entity.UserEntity;
 import ru.skypro.homework.service.AuthService;
+import ru.skypro.homework.transformer.UserTransformer;
 
 @Service
+@RequiredArgsConstructor
 public class AuthServiceImpl implements AuthService {
 
-    private final UserDetailsManager manager;
+    private final MyDatabaseUserDetailsService myDatabaseUserDetailsService;
```

Comment:
```suggestion
    private final UserDetailsManager myDatabaseUserDetailsService;
```
инъекцию надо делать всегда по интерфейсу, а не реализации